README.md

```diff
@@ -1,113 +1,117 @@
+## Getting Started
 
-# Getting Started
-Vulkan-Hpp is part of the LunarG Vulkan SDK since version 1.0.24. Just include ```<vulkan.hpp>``` and you're ready to use the C++ bindings. If you're using a Vulkan version not yet supported by the Vulkan SDK you can find the latest version of the header [here](https://github.com/KhronosGroup/Vulkan-Hpp/blob/master/vulkan/vulkan.hpp).
+Vulkan-Hpp is part of the LunarG Vulkan SDK since version 1.0.24. Just `#include <vulkan/vulkan.hpp>` and you're ready to use the C++ bindings. If you're using a Vulkan version not yet supported by the Vulkan SDK you can find the latest version of the header [here](https://github.com/KhronosGroup/Vulkan-Hpp/blob/master/vulkan/vulkan.hpp).
+
+### Minimum Requirements
 
-# Minimum Requirements
 Vulkan-Hpp requires a C++11 capable compiler to compile. The following compilers are known to work:
 * Visual Studio >=2015
 * GCC >= 4.8.2 (earlier version might work, but are untested)
 * Clang >= 3.3
 
-# namespace vk
+## Usage
+
+### namespace vk
+
 To avoid name collisions with the Vulkan C API the C++ bindings reside in the vk namespace. The following rules apply to the new naming
+
 * All functions, enums, handles, and structs have the Vk prefix removed. In addition to this the first leter of functions is lower case.
-  * ```vkCreateImage``` can be accessed as ```vk::createImage```
-  * ```VkImageTiling``` can be accessed as ```vk::ImageTiling```
-  * ```VkImageCreateInfo``` can be accessed as ```vk::ImageCreateInfo```
+  * `vkCreateImage` can be accessed as `vk::createImage`
+  * `VkImageTiling` can be accessed as `vk::ImageTiling`
+  * `VkImageCreateInfo` can be accessed as `vk::ImageCreateInfo`
 * Enums are mapped to scoped enums to provide compile time type safety. The names have been changed to 'e' + CamelCase with the VK_ prefix and type infix removed. In case the enum type is an extension the extension suffix has been removed from the enum values.
- In all other cases the extension suffix has not been removed.
-  * ```VK_IMAGETYPE_2D``` is now ```vk::ImageType::e2D```.
-  * ```VK_COLOR_SPACE_SRGB_NONLINEAR_KHR``` is now ```vk::ColorSpaceKHR::eSrgbNonlinear```.
-  *  ```VK_STRUCTURE_TYPE_PRESENT_INFO_KHR``` is now ```vk::StructureType::ePresentInfoKHR```.
-*  Flag bits are handled like scoped enums with the addition that the _BIT suffix has also been removed.
+
+In all other cases the extension suffix has not been removed.
+  * `VK_IMAGETYPE_2D` is now `vk::ImageType::e2D`.
+  * `VK_COLOR_SPACE_SRGB_NONLINEAR_KHR` is now `vk::ColorSpaceKHR::eSrgbNonlinear`.
+  *  `VK_STRUCTURE_TYPE_PRESENT_INFO_KHR` is now `vk::StructureType::ePresentInfoKHR`.
+* Flag bits are handled like scoped enums with the addition that the `_BIT` suffix has also been removed.
```

Comment:
The `_BIT` here was causing all subsequent formatting to fail in my Markdown editor, I don't know if it's standards compliant but you'll also notice github formats all subsequent removals with italic.